packages/main/src/BusyIndicator.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="{{classes.main}}">
```

Comment:
I think you should still use the controlData partial (or whatever it's called). We can remove it from everywhere later, but keep it for the moment